README.md

```diff
@@ -31,6 +31,7 @@ Misskey is a decentralized microblogging platform. Since it exists within the Fe
 ### Important points to read before installing
 
 - *Misskey* require a dedicated root domain, eg. `misskey.domain.tld`
+- Due to Cypress dependency, *Misskey* only works on 64-bit CPU machines.
```

Comment:
This should be added to the `DISCLAIMER.md` file. The README.md is generated automativally

Reply:
Ah ok - Thanks